pkg/command/context.go

```diff
@@ -572,21 +584,55 @@ func doCSPAuthAndUpdateContext(c *configtypes.Context, endpointType string) (cla
 	return claims, nil
 }
 
-func promptContextType() (ctxCreationType string, err error) {
+func promptContextType() (ctxCreationType ContextCreationType, err error) {
+	ctxCreationTypeStr := ""
 	promptOpts := getPromptOpts()
 	err = component.Prompt(
 		&component.PromptConfig{
 			Message: "Select context creation type",
-			Options: []string{ContextMissionControl, ContextApplicationEngine, ContextK8SClusterEndpoint, ContextLocalKubeconfig},
-			Default: ContextMissionControl,
+			Options: []string{string(ContextMissionControl), string(ContextApplicationEngine), string(ContextK8SClusterEndpoint), string(ContextLocalKubeconfig)},
+			Default: string(ContextMissionControl),
 		},
-		&ctxCreationType,
+		&ctxCreationTypeStr,
 		promptOpts...,
 	)
 	if err != nil {
 		return
 	}
-	return
+
+	return stringToContextCreationType(ctxCreationTypeStr), nil
+}
+
+func stringToContextCreationType(ctxCreationTypeStr string) ContextCreationType {
```

Comment:
This function is implemented as part of https://github.com/vmware-tanzu/tanzu-plugin-runtime/pull/112/files#diff-a51ca6e4ebbf11582e078a669e21c58bb8937b2840464358a059a7017314e63aR35

Which does not have a separate type called `ContextUnknown`. I will be updating this TPR dependency as part of https://github.com/vmware-tanzu/tanzu-cli/pull/524 but in a meanwhile can we not introduce a new Type `ContextUnknown` here and return empty string and update the code accordingly?

Reply:
Sure. Removed the `ContextUnknown` in the PR and used empty string instead